subsys/bluetooth/host/hci_core.c

```diff
@@ -4230,6 +4230,22 @@ static int le_init(void)
 
 #if defined(CONFIG_BT_SMP)
 	if (BT_FEAT_LE_PRIVACY(bt_dev.le.features)) {
+		struct bt_hci_cp_le_set_rpa_timeout *cp;
+		buf = bt_hci_cmd_create(BT_HCI_OP_LE_SET_RPA_TIMEOUT,
+					sizeof(*cp));
+		if (!buf) {
+			return -ENOBUFS;
+		}
+
+		cp = net_buf_add(buf, sizeof(*cp));
+
```

Comment:
You could remove this empty line since the it kind of belongs together with what follows